_posts/2023-05-23-GUIs-in-PyQt5.md

```diff
@@ -1,18 +1,18 @@
-# Ramin Honary
-
-  - Worked at Cross Compass since 2018
-  - Platform Team development, full stack
-  - Worked primarily on Manufacia
-  - Currently on working on ML-Platform
-
```

Comment:
@ramin-honary-xc It does not seem that you read my comment.
Please add this block at the header. Please replace text for title and excerpt with proper ones.
Tag feature is not yet available, but you can add some keywords for tags.

---
layout: single
title: (title of the post)
issue-date: 2023.06.15
author: Ramin Honary
tags: PyQt 
excerpt: Short description of the post
---